packages/xo-web/src/common/intl/messages.js

```diff
@@ -86,6 +86,10 @@ const messages = {
   installedCertificates: 'Installed certificates',
   expiry: 'Expiry',
   fingerprint: 'Fingerprint',
+  certificate: 'Certificate (PEM)',
+  certificateChain: 'Certificate chain (PEM)',
+  privateKey: 'Private key (PKCS#8)',
+  installNewCertificate: 'Install new certificate',
```

Comment:
Current behavior appear to replace the current certificate instead of adding a new one, we should make this explicit.